docs/templating/design_engine/add_new_design.md

```diff
@@ -54,7 +54,7 @@ Now, create templates for the two sites.
 Templates for the main site should be placed in `templates/themes/standard`,
 and templates for the campaign site in `templates/themes/summersale`.
 
-First, modify the built-in general [page layout](../templates/template_configuration.md#page-layout) template for the standard site in `templates/themes/standard/pagelayout.html.twig`
+First, modify the built-in general [page layout](pagelayout.html.twig`
```

Comment:
Broken link syntax. I'll try to fix it in #1770 with https://doc.ibexa.co/en/master/templating/templates/template_configuration/#page-layout